### PR TITLE
[PR] Fix package.json and lockfile dependency split

### PR DESCRIPTION
**Pull request by Nyumat**
Heroku Build is failing due to:

```
`npm ci` can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.
```
I added the engines to the package-json as recommended by: 
https://stackoverflow.com/questions/73201969/heroku-b-fail-installing-dependencies
https://stackoverflow.com/questions/72576874/i-can-build-locally-but-my-heroku-build-is-failing
Here's to hoping this fixes the issue! 😄

### CODE REVIEW

**Comment by Nyumat** [comment]
Interesting, still getting the errors after the hopeful "fix" will do more research

**Comment by Nyumat** [comment]
The issue has been found. 

https://github.com/Nyumat/TalkToBeavs/commit/3d744d9df783d4a69e48e5d8b6b20485b2177e3f#:~:text=%22react%2Drouter%2Ddom%22%3A%20%22%5E6.9.0%22%2C

TLDR; `react-router-dom` was added as a dependency in a commit a while back when we were iterating fast on the codebase so it went un-noticed. Now that I've removed that single line of code, the deployment succeeds. 🥲 

A learning experience for sure!
